### DISCUSSION TITLE
Will installing OhMyZsh alter my PATH?

### DISCUSSION THREAD

**Discussion by heafnerj** [ohmyzsh/ohmyzsh#10473] [Help] [answered]
I want to experiment with OhMyZsh on my Mac running Catalina. My .zshrc file currently contains PATH settings for Anaconda, perl, TeX Live, etc. Will installing OhMyZsh alter any of this?

Here's my current .zshrc file:

```
# >>> conda initialize >>>
# !! Contents within this block are managed by 'conda init' !!
__conda_setup="$('/Users/heafnerj/miniconda3/bin/conda' 'shell.zsh' 'hook' 2> /$
if [ $? -eq 0 ]; then
    eval "$__conda_setup"
else
    if [ -f "/Users/heafnerj/miniconda3/etc/profile.d/conda.sh" ]; then
        . "/Users/heafnerj/miniconda3/etc/profile.d/conda.sh"
    else
        export PATH="/Users/heafnerj/miniconda3/bin:$PATH"
    fi
fi
unset __conda_setup
# <<< conda initialize <<<

# Joe added these on 20200717 for Perl to use latexindent
#export PATH="/Users/heafnerj/perl5/perlbrew/bin:/Users/heafnerj/perl5/perlbrew$
source .bashrc
```

**Comment by rwmitchell** [comment, marked as answer]
Make a backup copy of your .zshrc file.  If it doesn't carry thru the conda path, add it back in.  Only the initial install mucks with .zshrc, after that, updates shouldn't change anything.

I'd add the ```source .bashrc``` near the bottom, but that may depend on what it contains.

**Reply by heafnerj** [reply]
Thank you.

**Reply by heafnerj** [reply]
Just FYI, I'm up and running!
